### ISSUE TITLE
Start handling CheCluster v2alpha1 in the che-operator

### ISSUE THREAD

**Issue by metlos** [eclipse-che/che#19453]
This is a subtask of #19448 and depends on #19449.

The `spec.che7` of the `CheCluster` `v2alpha1` is going to be identical to the current `CheCluster` `v1`. The che-operator should start to handle the `v2alpha1` version of the CR by merely looking at `spec.che7` and reconciling it the same way as `v1` is at the moment.

**Comment by metlos** [comment]
This is superseded by #19845. Once we get to the point where we need to define the new CR version, we can revisit this issue or create a new one.
